docs/source/changelog.rst

```diff
@@ -1,11 +1,44 @@
 Changelog
 =========
 
+0.2.2
+-----
+
+- Added missing import warnings to Aito client.
+- Updated AitoClient API documentation.
+
 0.2.1
 -----
 
-- upload_entries now accepts genetors as well as lists
-- upload_entries_by_batches has been deprecated
+- AitoClient upload_entries now accepts genetors as well as lists. Example usage:
+
+  .. code-block:: python
+
+    import os
+    from aito.sdk.aito_client import AitoClient
+
+    def example_generator(start, end):
+        for idx in range(start, end):
+            entry = {'id': idx}
+            yield entry
+
+    env_var = os.environ
+    aito_client = AitoClient(env_var['AITO_INSTANCE_URL'], env_var['AITO_API_KEY'])
+
+    schema = {"type": "table", "columns": {"id": {"nullable": True,"type": "Int"}}}
+
+    table_name = "table_name"
+
+    aito_client.create_table(table_name=table_name, table_schema=schema)
+
```

Comment:
```suggestion
```
I think we should remove this since the changelog should be concise the point is just to show how the upload entries can handle the generator

Reply:
Tommi asked for examples so... :sweat_smile:

Reply:
I think it should be in the doc, not the changelog